frontend/components/bc/BcMaintenanceBanner.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { warn } from 'vue'
+
+const { public: { maintenanceTS } } = useRuntimeConfig()
+const { tick } = useInterval(60)
+const { t: $t } = useI18n()
+
+const maintenanceLabel = computed(() => {
+  if (!maintenanceTS) {
+    return
+  }
+  const parsed = typeof maintenanceTS === 'number' ? maintenanceTS : parseInt(maintenanceTS)
```

Comment:
Maybe, the `? :` is not necessary, I imagine that this line would work the same way:
```TS
const parsed = Int(maintenanceTS)
```

Reply:
I don't think there is an `Int` constructor in JS. Do you mean Number?

Reply:
🤦🏻